### DISCUSSION TITLE
post/animations-on-the-web

### DISCUSSION THREAD

**Discussion by giscus** [bepyan/bepyan.me.v2#39] [General]
# post/animations-on-the-web

import { DynamicMail } from '~/components/examples/dynamic-mail'; import { Easing, Linear } from '~/components/examples/easing'; 들어가면서 최근 아주 흥미롭게 수강했던 강좌, An...

https://bepyan.me/post/animations-on-the-web

**Comment by chebread** [comment]
평안님 반갑습니다. 항상 좋은 글 감사드립니다! 프론트엔드 개발을 계속하다 보니, 에니메이션에 관심이 생겨서요. 에니메이션을 공부하려면, CSS 에니메이션을 자세히 공부를 해야하나요? 아니면 Frammer-motion, GSAP 같은 라이브러리를 학습해야 하나요? 요즘 1분코딩님의 에니메이션 강의도 찾아보고 있는데, 정확히 어떤 방향으로 공부할 지를 모르겠네요.. 조언 부탁드립니다!

**Reply by bepyan** [reply]
안녕하세요. 찾아오셔서 읽어 주셔서 감사합니다.
방법에는 정답이 없고 **꾸준히 공부하는 것**이 가장 중요한 것 같습니다. 

개인적인 조언을 드리자면, 아래 단계로 접근하는 것을 추천합니다.

1. 좋게 느꼈던 애니메이션을 기록하기 (링크든 영상이든)
2. 그 애니메이션이 어떻게 구현했는지 살펴보기, 찾아보기
3. 직접 그 애니메이션을 만들어 보기

애니메이션은 이론 보단 실전이니까요.

굳이 CSS와 라이브러리를 비교하자면, CSS 애니메이션으로 시작해보길 추천합니다. 라이브러리 내부에선 결국 CSS를 기반을 두기 때문입니다. 더 멀리 갈 수 있는 길인 것 같습니다.
